x-pack/plugins/cases/common/constants.ts

```diff
@@ -57,7 +76,19 @@ export const SUPPORTED_CONNECTORS = [
 export const MAX_ALERTS_PER_SUB_CASE = 5000;
 export const MAX_GENERATED_ALERTS_PER_SUB_CASE = 50;
 
+/**
+ * This must be the same value that the security solution plugin uses to define the case kind when it registers the
+ * feature for the 7.13 migration only.
+ *
+ * This variable is being also used by test files and mocks.
+ */
+export const SECURITY_SOLUTION_OWNER = 'securitySolution';
```

Comment:
this is what i feel should be an enum

Reply:
not necessary until we get observability, dont worry for now

Reply:
We are not planning to adding another `owner` variable inside cases. We don't need it. This is only for migration for saved object prior to RBAC and some tests.